From 549529eb36e21ab1e21c56ecf9c67e7db7f5a5f4 Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Sat, 23 Jan 2021 15:06:00 +0100
Subject: [PATCH] Revert "Disarm button on release of mouse for issue #527."

This reverts commit 331dbf8a42325f4bb83e1764b8460f08ff7f46b4
as it breaks application menu staying open on mouse click
---
 src/ybutton.cc | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/ybutton.cc b/src/ybutton.cc
index 1c11756d..8737b7b3 100644
--- a/src/ybutton.cc
+++ b/src/ybutton.cc
@@ -290,9 +290,6 @@ void YButton::handleButton(const XButtonEvent &button) {
         }
     }
     YWindow::handleButton(button);
-    if (fEnabled && button.type == ButtonRelease && button.button == Button1) {
-        setArmed(false, false);
-    }
 }
 
 void YButton::handleClick(const XButtonEvent &button, int count) {
-- 
2.30.0

